Makefile

```diff
@@ -24,7 +24,8 @@ build-all:
 	gox -verbose \
 	-ldflags "-X main.version=${VERSION}" \
 	-os="linux darwin windows " \
-	-arch="amd64 386" \
+	-arch="amd64 386 armv5 armv6 armv7 arm64" \
```

Comment:
Take a look at this [PR](https://github.com/Masterminds/glide/pull/528).
As explained there you must use a gox fork to work.